commune_app/tests/test_chores.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from commune_app import models
+from commune_app.all_models.users import User
+from commune_app.all_models.chores import Chore
```

Comment:
I think you can do it like:
```suggestion
from commune_app.models import User, Chore
```

Reply:
had to keep second line..
but the other 2 i turned them to one line.